includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -1,21 +1,185 @@
 <?php
 
 class DesignSystemApiController extends WikiaApiController {
+	private $wikiId;
+	private $lang;
+
+	private $data = [
+		'brand-logo' => [
+			'type' => 'brand-logo',
+			'href' => '#',
+			'elements' => [
+				[
+					'type' => 'line-image',
+					'image' => 'company/logo-fandom',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'global-navigation-logo-fandom',
+					]
+				],
+				[
+					'type' => 'line-image',
+					'image' => 'company/logo-wikia',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'global-navigation-logo-wikia',
+					]
+				],
+				[
+					'type' => 'translatable-text',
+					'key' => 'global-navigation-header-title',
+				],
+			],
+		],
+		'brand-links' => [
+			[
+				'type' => 'link-branded',
+				'brand' => 'tv',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'global-navigation-brandlink-vertical-tv'
+				],
+				'href' => 'http:\/\/tv.wikia.com'
+			],
+			[
+				'type' => 'link-branded',
+				'brand' => 'games',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'global-navigation-brandlink-vertical-games'
+				],
+				'href' => 'http:\/\/games.wikia.com'
+			],
+			[
+				'type' => 'link-branded',
+				'brand' => 'movies',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'global-navigation-brandlink-vertical-movies'
+				],
+				'href' => 'http:\/\/movies.wikia.com'
+			],
+			[
+				'type' => 'links-branded',
```

Comment:
There is no `href` field inside this object. Is that a mistake or should we think about simple branded element?